### PR TITLE
Enable proxyMiddleware to output more information.

### PR DESCRIPTION
**Pull request by thegatheringstorm**
:smile:

### CODE REVIEW

**Comment by MaxMilton** [comment]
I like this but I feel it should be optional because on some projects it can flood your terminal with excessive information.

File: template/build/dev-server.js [hunk 1]
```diff
@@ -47,6 +47,7 @@ Object.keys(proxyTable).forEach(function (context) {
   if (typeof options === 'string') {
     options = { target: options }
   }
+  options.logLevel = 'debug'
```
Comment (Aladdin-ADD) [inline template/build/dev-server.js]: can we add a comment, so users can have a choice to turn off?

**Comment by Aladdin-ADD** [comment]
@MaxMilton  I'm fine with the the current behavior. maybe we can just add a comment, letting users have a choice to turn off?

**Comment by MaxMilton** [comment]
That sounds like a decent way to do things. :+1:

**Comment by thegatheringstorm** [comment]
Thanks!

**Pull request closed without merging**